compiler/luci/service/src/Nodes/CircleHardSwish.test.cpp

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "luci/Service/CircleNodeClone.h"
+
+#include <gtest/gtest.h>
+
+TEST(CloneNodeTest, clone_HardSwish)
+{
+  auto g = loco::make_graph();
+  auto node_hardswish = g->nodes()->create<luci::CircleHardSwish>();
+
+  auto gc = loco::make_graph();
+  auto cloned = luci::clone_node(node_hardswish, gc.get());
+  ASSERT_NE(nullptr, cloned);
+  ASSERT_EQ(gc.get(), cloned->graph());
+
+  auto cloned_hardswish = dynamic_cast<luci::CircleHardSwish *>(cloned);
+  ASSERT_NE(nullptr, cloned_hardswish);
+}
```

Comment:
Please refer `compiler/luci/service/src/Nodes/CircleRelu.test.cpp` and add shape and dtype inference test too.

Reply:
Ok, I will check the file.
=)

Reply:
I've added shape and dtype inference test for HardSwish.
- commit id: ccde26a